pkg/controller/prpqr_reconciler/prpqr_reconciler.go

```diff
@@ -157,44 +165,88 @@ func (r *reconciler) reconcile(ctx context.Context, req reconcile.Request, logge
 		statusByJobName[jobName] = &prpqr.Status.Jobs[i]
 	}
 
+	var prowjobs []*prowv1.ProwJob
+
 	baseMetadata := metadataFromPullRequestUnderTest(prpqr.Spec.PullRequest)
 	for _, jobSpec := range prpqr.Spec.Jobs.Jobs {
 		mimickedJob := jobSpec.JobName(jobconfig.PeriodicPrefix)
-		logger = logger.WithFields(logrus.Fields{"want-job": mimickedJob})
+		jobSpecWithUID := jobSpecWithUID{ReleaseJobSpec: &jobSpec}
+
+		if jobSpec.AggregatedCount > 0 {
+			jobSpecWithUID.UID = uuid.NewV1().String()
```

Comment:
Could we make the aggregation UID by hashing together the name of the PRPQR and the name of the aggreated job? That way it would be deterministic for the given prpqr+job and still unique.